src/olympia/versions/tests/test_admin.py

```diff
@@ -0,0 +1,23 @@
+from pyquery import PyQuery as pq
+
+from olympia.amo.tests import TestCase, addon_factory, user_factory
+from olympia.amo.urlresolvers import reverse
+
+
+class TestVersionAdmin(TestCase):
+
+    def test_detail_view_has_download_link(self):
+        addon = addon_factory()
+        version = addon.current_version
+        detail_url = reverse(
+            'admin:versions_version_change', args=(version.pk,)
+        )
+        user = user_factory()
+        self.grant_permission(user, 'Admin:Tools')
+        self.grant_permission(user, 'Admin:Advanced')
```

Comment:
Can we have a separate call before the actual "200 response" to double-check that just doing `self.client.get(detail_url...)` without `Admin:Advanced` leads to an error response? (Should be 403)

Reply:
Done in f6b3e29559.